src/soundbase.cpp

```diff
@@ -41,6 +41,8 @@ CSoundBase::CSoundBase ( const QString& strNewSystemDriverTechniqueName,
     lNumDevs          = 1;
     strDriverNames[0] = strSystemDriverTechniqueName;
 
+    setObjectName ( "CSoundBase::" + strSystemDriverTechniqueName );
```

Comment:
Do we see this as a separate thread under linux? I thought you will only see the Jack audio thread?

Reply:
Isn't this thread created from linux/sound.cpp?  This then should say "CSoundBase::jack" or whatever gets passed in.  Client-side isn't currently so interesting, though.

Reply:
Yes, it's client only. We should focus on the server for now. So no need to set an object name here.